BLSQ/iaso
# POLIO-1229: Track dashboards use with analytics

Explain what problem this PR is resolving
- To be able to track dashboards use with analytics
Related JIRA tickets : [POLIO-1229](https://bluesquare.atlassian.net/browse/POLIO-1229)

## Self proofreading checklist


## Changes
- Add new field in pages DB
- Add new field on the page form
- Add the field value in all pages in view.py

## How to test
- Edit or create a page (dashboard)  and add a script from [plausible](https://plausible.io) for example
- Open in view mode the page and inspect the html code and check if the script is there in the head tag

## Print screen / video
[Screencast from 2023-12-12 10-23-18.webm](https://github.com/BLSQ/iaso/assets/19631540/23b12b0a-d5b3-45a8-ae82-d943babe99b0)



[POLIO-1229]: https://bluesquare.atlassian.net/browse/POLIO-1229?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
